owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/history/model/atomic/HistorySubjectOrganization.java

```diff
@@ -0,0 +1,7 @@
+package fr.openwide.core.jpa.more.business.history.model.atomic;
+
+public enum HistorySubjectOrganization {
+	
+	FAFCEA
```

Comment:
Removed. I don't see the point in adding it to the basic app as I prefer to let people define the right context information in the project.